.env.sample

```
@@ -12,4 +12,9 @@ KAFKA_UI_PORT=
 
 SPRING_PROFILES_ACTIVE=
 
-OPENAI_API_KEY=
\ No newline at end of file
+NAVER_MAP_CLIENT_ID=
+NAVER_MAP_CLIENT_SECRET=
+
+OPENAI_API_KEY=
+
+SLACK_BOT_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)